packages/blocks/src/columns-block/index.ts

```diff
@@ -0,0 +1,12 @@
+import type { ColumnsBlockModel } from './columns-model.js';
+
+export * from './columns-block.js';
+export * from './columns-model.js';
+
+declare global {
+  namespace BlockSuite {
+    interface BlockModels {
+      'affine:columns': ColumnsBlockModel;
```

Comment:
The flavour is supposed to be `affine:column` and make it `blocks/src/column`, instead of using the plural term of "columns"  🤔

In slash menu, keeping the "Columns" entry text for user looks fine.